test/controllers/ViewArrivalsControllerSpec.scala

```diff
@@ -43,23 +43,24 @@ import scala.concurrent.Future
 
 class ViewArrivalsControllerSpec extends SpecBase with MockitoSugar with JsonMatchers with ModelGenerators with NunjucksSupport with BeforeAndAfter {
 
-  private val mockDestinationConnector          = mock[DestinationConnector]
+  private val mockArrivalMovementConnector      = mock[ArrivalMovementConnector]
   private val mockCustomOfficeConversionService = mock[ViewMovementConversionService]
```

Comment:
why are we mocking the conversionService this feels a little heavy handed and may give us a false positive

Reply:
I think it's fine the usage here since it allows us to focus on the behaviour of the controller. The `ViewMovementConversionService` should be unit tested and it provides a simple interface so I think this is fine?